src/libs/routes/validateHandler.ts

```diff
@@ -0,0 +1,49 @@
+import validateHelper from "./validateHelper";
+export default function validateHandler(ObjData) {
+  return function(req, res, next) {
+    const keys = Object.keys(ObjData);
+    console.log("The keys are", keys);
+    keys.forEach(function(key) {
+      const item = ObjData[key];
+      const value = item.in.map(function(items) {
+        return req[items][key];
+      });
+      console.log("The value is", value);
+      console.log(value);
+      const validatedValue = value.filter(item => item);
+      if (item && item.required) {
+        if (validatedValue.length !== value.length) {
+          next({
+            error: "Not Valid",
+            message: `${key} is required`,
+            status: 404
+          });
+        }
+      }
+      if (!Array.isArray(validatedValue[0])) {
```

Comment:
instead of getting the 0th element of an array like this validatedValue[0] every time, get the element and store it to another variable

Reply:
if i use only as a variable then i have errors while comparing validatedValue.length and value.length
which i used for checking required validation

Reply:
i fixed it and pushed it on feature/40010 branch